noisyatom/config/settings.py

```diff
@@ -72,11 +72,17 @@
         'django.contrib.messages',
         'django.contrib.staticfiles',
         'django.contrib.sites',
-        'account',
-        'catalog',
+
+        'account.apps.AccountConfig',
+        'blogs.apps.BlogsConfig',
+        'catalog.apps.CatalogConfig',
         'cms.apps.CmsConfig',
         'qrcode',
         'ee_test',
+
```

Comment:
Remove space.

Reply:
we need it to put the separate between project applications , django automatic codes and third party library